src/java/org/apache/cassandra/schema/TableParams.java

```diff
@@ -614,6 +630,7 @@ public static class Serializer implements MetadataSerializer<TableParams>
         public void serialize(TableParams t, DataOutputPlus out, Version version) throws IOException
         {
             out.writeUTF(t.comment);
+            out.writeUTF(t.securityLabel);
```

Comment:
I do not think we can do this. You should move this to the very end of this method and do version check.